### DISCUSSION TITLE
MgGraph powershell connection with certificate results in no scopes

### DISCUSSION THREAD

**Discussion by oden08** [microsoftgraph/msgraph-sdk-powershell#2258] [Q&A]
I have created a app registration in Entra to connect to MgGraph without login prompt (working on replacing MsolUser powershell scripts for azure ad connect license management). I am able to connect with my clientid / tenantid / certificatethumbprint but I have no permissions and Get-MgContext results in an empty "scopes" field. Can anyone point me in the direction of why I wouldn't be granted the permissions that I set in "API Permissions"?

**Comment by RayGHeld** [comment]
@oden08 When using the certificate (the client_credentials grant flow) you can only get application permissions in that flow.  Please ensure that you have application permissions configured and then the scope for the connect-mggraph should be "https://graph.microsoft.com/.default"
